mne/io/brainvision/brainvision.py

```diff
@@ -195,7 +195,7 @@ def _read_vmrk_events(fname, event_id=None, response_trig_shift=0):
     for info in items:
         mtype, mdesc, onset, duration = info.split(',')[:4]
         onset = int(onset)
-        duration = int(duration)
+        duration = (int(duration) if duration.isdigit() else 0)
```

Comment:
Do you know if `isdigit` will be robust enough?
```
>>> '100 '.isdigit()
False
>>> int('100 ')
100
```
In other words, if there is whitespace there could be a problem. Not sure if there are other problematic cases. I wonder if there's a safer check?